roles/sp_client_policy/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
No, you need to make it so this role is reusable like the other role. Can you take the template from the other role and just change to the relevant options